### PR TITLE
Make MaxConcurrentReconciles configurable in function-controller

### PR DESCRIPTION
**Pull request by aerfio**
**Description**

Changes proposed in this pull request:

- as in title, for now only for function-controller

### CODE REVIEW

File: resources/serverless/values.yaml [hunk 1]
```diff
@@ -136,7 +136,9 @@ containers:
       functionBuildExecutorImage:
         value: gcr.io/kaniko-project/executor:v0.22.0
       functionBuildRepoFetcherImage:
-        value: eu.gcr.io/kyma-project/function-build-init:a8a6bca9
+        value: eu.gcr.io/kyma-project/function-build-init:PR-9496
```
Comment (dbadura) [inline resources/serverless/values.yaml:139]: Why did you change this image?

Comment (aerfio) [inline resources/serverless/values.yaml:139]:
```suggestion
        value: eu.gcr.io/kyma-project/function-build-init:PR-9800
```

**Review by dbadura — APPROVED** [review]
